content/technotes/2017-11-08-solrium-solr-r.md

```diff
@@ -0,0 +1,278 @@
+---
+slug: "solrium-solr-r"
+title: "solrium 1.0: Working with Solr from R"
+date: 2017-11-08
+authors:
+  - name: Scott Chamberlain
+categories:
+  - technotes
+topicid:
+tags:
+- data
+- solr
+- solrium
+- databases
+- R
+---
+
+
+
+Nearly 4 years ago I wrote on this blog about an R package [solr][solrpkg] for working with the database [Solr][solr]. Since then we've created a refresh of that package in the [solrium][] package. Since `solrium` first hit CRAN about two years ago, users have raised a number of issues that required breaking changes. Thus, this blog post about a major version bump in `solrium`.
+
+<br>
+
+## What is Solr?
+
+Solr is a "search platform" - a NoSQL database - data is organized by so called documents that are xml/json/etc blobs of text. Documents are nested within either collections or cores (depending on the mode you start Solr in).
+
+Solr makes it easy to search for documents, with a huge variety of parameters, and a number of different data formats (json/xml/csv).
+
+Solr is similar to [Elasticsearch][es] (see our Elasticsearch client [elastic][]) - and was around before it. Solr in my opinion is harder to setup than Elasticsearch. Both have very nice APIs though.
```

Comment:
reason to use Solr vs Elasticsearch? 

Maybe you don't need to answer in a technote but this Q came to mind after reading this.

Reply:
maybe put above 3 sentences into one para - similar content

Reply:
Yeah, i can't really address which to choose, and i think a bit out of scope here